mycroft/configuration/mycroft.conf

```diff
@@ -376,10 +405,8 @@
     // ('mycroft_mark_1', 'picroft', 'mycroft_mark_2pi')
     // to disable forced ntp_sync in official mycroft platforms
     // set this to false
-    "force_mycroft_ntp": true,
+    "force_mycroft_ntp": true
 
-    // Run a self test at bootup?
-    "test": false
```

Comment:
0¿o;

Reply:
mk1 specific code, in the process of being moved https://github.com/OpenVoiceOS/ovos_phal_mk1
the test is related to the arduino only